src/languages/es.js

```diff
@@ -346,11 +346,13 @@ export default {
         settledPaypalMe: 'Pagado con PayPal.me',
         settleExpensify: 'Pagar con Expensify',
         settleElsewhere: 'Voy a pagar de otra forma',
+        payExpenseElsewhere: 'Pagar de otra forma',
         settlePaypalMe: 'Pagar con PayPal.me',
         requestAmount: ({amount}) => `solicitar ${amount}`,
         splitAmount: ({amount}) => `dividir ${amount}`,
         amountEach: ({amount}) => `${amount} cada uno`,
         payerOwesAmount: ({payer, amount}) => `${payer} debe ${amount}`,
+        payerPaidAmount: ({payer, amount}) => `${payer} pagado ${amount}`,
         payerSettled: ({amount}) => `pagado ${amount}`,
```

Comment:
I need to see as screenshot for this one because it really depends on the context.

Reply:
Actually I found out this one is only used as a fallback if the message is malformed or unavailable (should not happen). However it should be here in the chat where you see the `paid $10.00`. So I think this is probably not important

<img width="878" alt="image" src="https://github.com/Expensify/App/assets/36083550/5e4705ca-a87c-4e63-b5ad-22dee61a6de0">

Reply:
Awesome, then:
```suggestion
        payerSettled: ({amount}) => `pagó ${amount}`,
```

Reply:
Thanks!